automount-actions/scripts/bin/auto-mount.sh

```diff
@@ -57,40 +57,29 @@ try_mount() {
 }
 
 # cleanup acts to ensure this daemon remains tidy
-cleanup () {
+cleanup() {
     _exit_status=$?
+
     sync
     
-    # Use find instead of ls for better reliability and portability
-    # This handles filenames with spaces and special characters properly
-    if [ -d "$MOUNT_DIR" ]; then
-        # Find all directories in MOUNT_DIR (these are our UUID mount points)
-        # Use -print0 and read -d '' to handle filenames with spaces/newlines
-        find "$MOUNT_DIR" -mindepth 1 -maxdepth 1 -type d -print0 | \
-        while IFS= read -r -d '' _mount_path; do
+    # Check if MOUNT_DIR exists and continue with our life
+    [ ! -e "$MOUNT_DIR" ] || {
+        for _mount_path in "$MOUNT_DIR"/*; do
+            # Skip if glob didn't match anything
+            [ -e "$_mount_path" ] || continue
+            
             _uuid=$(basename "$_mount_path")
             
             # Attempt to unmount the device
-            if umount "$_mount_path" 2>/dev/null; then
-                echo "Unmounted: $_mount_path"
-            else
-                echo "WARN: Failed to unmount $_mount_path"
-                # Continue trying to clean up other mounts
-            fi
+            umount "$_mount_path" 2>/dev/null && echo "Unmounted: $_mount_path" || echo "WARN: Failed to unmount $_mount_path"
```

Comment:
Linewrap at least 120, preferably 80.

```suggestion
        umount "$_mount_path 2>/dev/null &&
        echo "Unmounted: $_mount_path"   ||
        echo "WARN: Failed to unmount $_mount_path"
```

Reply:
+1